Default.sublime-keymap

```diff
@@ -511,6 +511,45 @@
         ],
         "args": { "action": "view" }
     },
+    {
+        "keys": ["m"],
+        "command": "gs_log_graph_toggle_more_info",
+        "context": [
+            { "key": "setting.git_savvy.log_graph_view", "operator": "equal", "operand": true }
+        ]
+    },
+    {
+        "keys": ["n"],
+        "command": "gs_log_graph_next_commit",
+        "args": {"forward": true},
+        "context": [
+            { "key": "setting.git_savvy.log_graph_view", "operator": "equal", "operand": true }
+        ]
+    },
+    {
+        "keys": ["p"],
+        "command": "gs_log_graph_next_commit",
+        "args": {"forward": false},
+        "context": [
+            { "key": "setting.git_savvy.log_graph_view", "operator": "equal", "operand": true }
+        ]
+    },
+    {
+        "keys": ["down"],
```

Comment:
Generally not okay with modifying behavior of arrow keys.  Elsewhere, we're using `,` and `.` to navigate within a view.  Please modify the above to correspond with those keys, and remove the down/up bindings.  Thanks!